Assets/_BForBoss/_Core/Prefabs/EditorOnly/ToolsMenu - EditorOnly.prefab

```diff
@@ -0,0 +1,45 @@
+%YAML 1.1
```

Comment:
Is this needed here?

Reply:
It is cleaner like this I think, The Prefab has the `Editor Only` tag on it so that when we build this prefab will be excluded. So going forward we could attach any Editor only Tools and Debugs can be attached to this Prefab

Reply:
Should this prefab be under `Utility`?

Reply:
![image](https://user-images.githubusercontent.com/5779541/159176976-73f7c618-5b5a-4c85-9422-a4a65084ed19.png)
How would you differentiate between the 2?

Reply:
![image](https://user-images.githubusercontent.com/23490604/159177427-8cc6f746-f3ac-4205-ad43-ec570edeab74.png)

The Editor Only tag ensures that when the games builds, the Prefab is not built

Reply:
So if we were to add a new tool, how would you choose where to put it under?

Reply:
> ![image](https://user-images.githubusercontent.com/5779541/159177982-c96df763-2737-4a7e-8236-fee3aec3fa52.png) And after the creation failed, there's these Git changes that shouldn't really happen. I'm assuming it's related to creating the asset? but it should've been taken care of in this PR (Having an asset)

Yeah it should be handled with my latest commit, I was having this issue with Resources.LoadAll<> not Resources.Load